### ISSUE TITLE
If infinite loop is set to false, all slides have "aria-hidden" set to true

### ISSUE THREAD

**Issue by esr360** [kenwheeler/slick#1408]
If I set the "infinite" property to "false", the "aria-hidden" attribute on every slide seems to be set to true, when naturally the visible slides should be set to false if the slide is visible.

http://codepen.io/esr360/pen/XbMpKg

**Comment by leggomuhgreggo** [comment]
Looks like its a bug with center mode. I tried disabling that and it worked.

http://codepen.io/gwst/pen/dovNVg

If you need to use center mode and want to hack proper aria function until this get's fixed, I think you can get it done by using the beforeChange event to toggle the attribute. [More here](https://github.com/kenwheeler/slick#events)

**Comment by esr360** [comment]
Hi thanks for your reply and for the hack until there's a fix.

Just to point out, if I remove the "infinite" attribute from the above example, but leave centerMode set to "true", everything works as expected, so to me it seems like an issue with "infinite" and not "centerMode" - but honestly I have no idea :)

**Comment by gunnx** [comment]
Also having this issue on v1.5.0 
To fix, I'm adding code to the afterchange event to set the active slide to aria-hidden false

**Comment by thiemeljiri** [comment]
I'm having the same issue. This is a critical bug as far as accessibility is concerned. It's like if the whole carousel wasn't present.

**Comment by charlesteets** [comment]
I worked around this in my local branch _**FOR MY PARTICULAR USE CASE.**_ In `Slick.prototype.setSlideClasses`, by creating a new logic path. There's a  `if (_.options.centerMode === true)` check, and within that a `if (_.options.infinite === true)` check, but there's no `else` for the infinite option check.

All I did was copy everything within `if (_.options.infinite === true)` into its own `else` (this is the same as removing the line with the `if` check on it).  

My use case is these particular settings

arrows: true,
slidesToShow: 1,
infinite: false,
centerMode: true,
slidesToScroll: 1,
focusOnChange: true,

and I am sure that this hack does not universally fix this issue but I didn't have time to deep dive into this chunk of logic and see exactly what was going on, and what would need to be different for infinite mode on/off.

**Comment by richgerdes** [comment]
I've opened PR #4307 for this. I copied the logic as suggested by @underclockeddev and adjusted the code to remove what I think was needed for the infinite loop to render correctly. Hopefully this addresses the bug.
